### DISCUSSION TITLE
Is there any plan to support opengles?

### DISCUSSION THREAD

**Discussion by dangyajun** [MethanePowered/MethaneKit#134] [Q&A] [answered]

**Comment by egorodet** [comment, marked as answer]
Short answer is no. Methane Kit is designed to support only modern graphic APIs, like DirectX 12, Vulkan and Metal, which allow to enable  most of platforms support including desktop and mobile. This APIs allow to achieve maximum performance on modern hardware,  OpenGL ES is considered as outdated and legacy graphics API targeted at mobiles, also it is simple enough to use, meaning that it does not require any additional wrapper APIs.

I wonder what is you particular use case and platform, which requires support of OpenGL ES?

**Reply by dangyajun** [reply]
In order to support some old devices, such as Raspberry Pi 1, etc.
